openmdao/core/driver.py

```diff
@@ -823,10 +872,22 @@ def run(self):
         bool
             Failure flag; True if failed to converge, False is successful.
         """
+        self._start_time = time.time()
+
         with RecordingDebugging(self._get_name(), self.iter_count, self):
             self._problem().model.run_solve_nonlinear()
 
         self.iter_count += 1
+
+        # save results of run
+        self.opt_result = {
+            'runtime': time.time() - self._start_time,
+            'iter_count': self.iter_count,
+            'obj_calls': self.get_driver_objective_calls(),
+            'deriv_calls': self.get_driver_derivative_calls(),
+            'exit_status': self.get_exit_status()
+        }
```

Comment:
These values aren't sticking when I run this on the `ex_brachistochrone.py` script in [dymos/examples/brachistochrone/test/ex_brachistochrone.py](https://github.com/OpenMDAO/dymos/blob/master/dymos/examples/brachistochrone/test/ex_brachistochrone.py)

```
Number of driver iterations: | 1
-- | --
Number of objective calls: | 1
Number of derivative calls: | 1
Execution start time: | 2022-07-15 12:23:26
Wall clock run time: | 00 hours 00 minutes 00 seconds 0.0 milliseconds
Exit status: | NOTRUN
```